main.go

```diff
@@ -333,14 +353,26 @@ func main() {
 		&cli.StringFlag{
 			Name:  "blockstore",
 			Usage: "specify blockstore parameters",
-			Value: cfg.NodeConfig.BlockstoreDir,
+			Value: cfg.NodeConfig.Blockstore,
 		},
 		&cli.BoolFlag{
 			Name:  "write-log-truncate",
+			Usage: "enables log truncating",
 			Value: cfg.NodeConfig.WriteLogTruncate,
 		},
+		&cli.BoolFlag{
+			Name:  "write-log-flush",
+			Usage: "enable hard flushing blockstore",
+			Value: cfg.NodeConfig.HardFlushWriteLog,
+		},
+		&cli.BoolFlag{
+			Name:  "no-blockstore-cache",
+			Usage: "disable blockstore caching",
```

Comment:
Same comment about caching above

Reply:
if true, the blockstore will not be wrapped in a caching layer